compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -1781,6 +1781,63 @@ std::optional<VectorizationTileSizes> inferSizesFromIR(linalg::PackOp op) {
   return result;
 }
 
+std::optional<SizesAndScalableFlags>
+getVectorInputSizesFromDestTiles(linalg::UnPackOp op,
+                                 ArrayRef<int64_t> vectorSizes,
+                                 ArrayRef<bool> scalableFlags) {
+  assert(vectorSizes.size() == op.getDestRank());
+  if (llvm::any_of(scalableFlags, [](bool val) { return val == false; })) {
+    return std::nullopt;
+  }
```

Comment:
I feel that I might be confusing myself here, but shouldn't we check for instances of `true` instead?
```suggestion
  if (llvm::any_of(scalableFlags, [](bool val) { return val == true; })) {
    return std::nullopt;
  }
```

Reply:
Yes, you are right and good catch!